plugins/modules/logstash_plugin.py

```diff
@@ -49,7 +49,7 @@
   version:
     type: str
     description:
-      - Specify plugin Version of the plugin to install. If plugin exists with previous version, it will NOT be updated.
+      - Specify version of the plugin to install. If plugin exists with previous version, it is NOT updated.
```

Comment:
`updated` → `update`. Also how about using `B(...)` instead upper-case text?
```suggestion
      - Specify version of the plugin to install. If plugin exists with previous version, it is B(not) update.
```

Reply:
Oops, sorry for the forgotten d :D